Formula/skim.rb

```diff
@@ -0,0 +1,32 @@
+class Skim < Formula
+  desc "Fuzzy Finder in rust!"
+  homepage "https://github.com/lotabout/skim"
+  url "https://github.com/lotabout/skim/archive/v0.5.2.tar.gz"
+  sha256 "41280bee2138afefff95f76a640b753d1cb9215e8391ef37eab7ccb3517d9019"
+
+  depends_on "rust" => :build
+
+  def install
+    (buildpath/"src/github.com/lotabout").mkpath
+    ln_s buildpath, buildpath/"src/github.com/lotabout/skim"
+    system "cargo", "install", "--root", prefix, "--path", "."
+
+    prefix.install "install"
+    (prefix/"shell").install %w[bash zsh].map { |s| "shell/key-bindings.#{s}" }
```

Comment:
Neither is `prefix/"shell"`

Reply:
@fxcoudert I actually followed fzf's formula: https://github.com/Homebrew/homebrew-core/blob/master/Formula/fzf.rb

Could you please tell me what's the standard location for storing shell bindings and vim plugins?

Reply:
https://github.com/Homebrew/homebrew-core/blob/8d0d13e425ea70d42d9597cd90e38e8949150329/Formula/interactive-rebase-tool.rb#L21-L49